examples/webgl_pmrem_test.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js PMREM directional light test</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+
+		<div id="container">
+			<div id="info">
+				<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> -
+                PMREM directional light test <a href="https://github.com/elalish" target="_blank" rel="noopener">Emmett Lalish</a>
+                <br>Top row is white metal
+                <br>Middle row is white dielectric
+                <br>Last row is black dielectric.
+                <br>Mouse-out is a standard Directional Light.
+                <br>Mouse-over is a PMREM of the skybox: a single bright pixel representing the same directional light source.
+                <br>The difference between these renders indicates the error in the PMREM approximations.
+			</div>
+		</div>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+            import { OrbitControls } from './jsm/controls/OrbitControls.js';
+            import { RGBELoader } from './jsm/loaders/RGBELoader.js';
+			import { PMREMGenerator } from './jsm/pmrem/PMREMGenerator.js';
+
+            var scene, camera, controls, renderer;
+            
+			function init() {
+
+				var width = window.innerWidth;
+				var height = window.innerHeight;
+				var aspect = width / height;
+
+				// renderer
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setSize( width, height );
+                renderer.setPixelRatio( window.devicePixelRatio );
+                renderer.gammaOutput = true;
+                renderer.physicallyCorrectLights = true;
+				document.body.appendChild( renderer.domElement );
+
+                window.addEventListener( 'resize', onResize, false );
+                
+				// scene
+
+				scene = new THREE.Scene();
+
+				// camera
+
+				camera = new THREE.PerspectiveCamera( 40, aspect, 1, 30 );
+                camera.position.set( 0, 0, 18 );
+
+                // controls
+                
+                controls = new OrbitControls( camera, renderer.domElement );
+                controls.update();
+
+                // light
+                
+                var directionalLight = new THREE.DirectionalLight( 0xffffff, 1 );
+                var x = 597;
+                var y = 213;
+                var theta = ( x + 0.5 ) * Math.PI / 512;
+                var phi = ( y + 0.5 ) * Math.PI / 512;
+
+                directionalLight.position.set(
+                    - Math.sin( phi ) * Math.cos( theta ),
+                    Math.cos( phi ),
+                    - Math.sin( phi ) * Math.sin( theta ) );
+
+                scene.add( directionalLight );
```

Comment:
Optional check: It looks correct.
```js
directionalLight.position.set(
    - Math.sin( phi ) * Math.cos( theta ),
    Math.cos( phi ),
    - Math.sin( phi ) * Math.sin( theta ) ).multiplyScalar( 100 );

scene.add( directionalLight );

scene.add( new THREE.DirectionalLightHelper( directionalLight ) );
```

Reply:
Instead of doing this:
```js
directionalLight.position.set(
    - Math.sin( phi ) * Math.cos( theta ),
    Math.cos( phi ),
    - Math.sin( phi ) * Math.sin( theta ) ));
``` 
how about
```js
directionalLight.position.setFromSphericalCoords( 1, - phi, theta );
```